packages/sit-onyx/src/components/OnyxTable/OnyxTable.ct.tsx

```diff
@@ -55,4 +55,22 @@ test.describe("Screenshot tests", () => {
       </OnyxTable>
     ),
   });
+
+  executeMatrixScreenshotTest({
+    name: "Table (hover styles)",
+    columns: ["default", "striped"],
+    rows: ["row-hover", "column-hover"],
+    // TODO: remove when contrast issues are fixed in https://github.com/SchwarzIT/onyx/issues/410
+    disabledAccessibilityRules: ["color-contrast"],
```

Comment:
fyi in dark mode the hover color is nearly invisible. maybe point that out to Jannick before we go beta?

Reply:
would be fine to add this info e.g. as checkbox to #410 I think :)